### DISCUSSION TITLE
Will API Requests update to include additional game stats that are now being collected

### DISCUSSION THREAD

**Discussion by Naveen512345** [Noxcrew/mcchampionship-api#12] [General]
Hi whoever manages the API requests from Noxcrew! I've been directed from the MCC discord server modmail to ask this question here, I was just wondering if the Noxcrew team were ever considering publicly sharing these other stats that they now collect for MCC games? For example I'd be really interested to see railtracks placed and other stats for RRR. I'm part of the power rankings team and we manually collect a lot of stats to improve our player ratings (including damage stats) so if it isn't something Noxcrew are opposed to sharing (or if it isn't too complicated to) that would be amazing for the MCC stats community! Even stats like sand collected, shots hit in Battle Box and others would be really cool to see if they're able to be shared through the API!

**Comment by MagicA550** [comment]
Hi there,

The short answer is yes, we want to expose more stats in the API. It's likely that, when this day comes, we'll have an API similar to the [MCC Island GraphQL API](https://github.com/noxcrew/mccisland-api). This would mark a natural progression for the API to its "v2" (final) state. At the moment, it's more of a question of "when" rather than "if".

I hope this provides more clarity - feel free to reach out again if you have further questions!

**Reply by Naveen512345** [reply]
Thank you for the response! We just realised that you guys have also removed damage death recaps for Sky Battle, would that be stats you guys are considering to share to the community also? Also would you happen to know an estimate for when v2 may come out, and would you guys ever consider sharing like an ugly CSV file or excel of the data you guys collect for each game/event (maybe here on the Github page) for stats people to use in the mean time until damage recaps come back or v2 gets released?

If that's possible that'd be so so helpful!
